app/controllers/api/v1/search_controller.rb

```diff
@@ -557,24 +565,48 @@ def self.generate_mongo_query_by_context(terms:, base_studies:, accessions:, que
           results_matched_by_data['numResults:scp'] = studies.length # Total number of SCP results
           # Azul study results to be added with SCP-4202
 
-          return {:studies => studies, :results_matched_by_data => results_matched_by_data}
```

Comment:
What's the difference between doing` {:variable => value}` vs `{variable: value}` and how did you know to use which one where lol

Reply:
They're functionally equivalent - `{variable: value}` is a linting suggestion from Rubocop.  It's just fewer characters is all.